operator/builtin/input/file/config.go

```diff
@@ -138,24 +142,36 @@ func (c InputConfig) Build(context operator.BuildContext) ([]operator.Operator,
 		filePathField = entry.NewAttributeField("file_path")
 	}
 
+	fileNameResolvedField := entry.NewNilField()
+	if c.IncludeFileNameResolved {
+		fileNameResolvedField = entry.NewAttributeField("file_name_resolved")
```

Comment:
I think we should look into standardizing file related attribute names as Otel semantic conventions.

Reply:
valid point. Are they any ideas for that? Should it be added to https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions?

Reply:
Yes. I think we need to decide on how we group this attributes, perhaps introduce a `file.* namespace and put everything there. Perhaps also some of the attributes discussed [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/2628) should in that namespace, e.g. `stream`?

Reply:
I suggest we do the following:
- In this PR, choose attributes names we think are likely to be appropriate. Update this PR accordingly. Expect that they may have to change.
- File a PR against the spec repo, which proposes these names as semantic conventions.

Does this seem like a reasonable approach? 

--- 

Am I correct in thinking that there aren't currently any semantic conventions for log attributes, so this would also entail establishing a new section in the spec repo? Possibly parallel to [here](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions)?

Reply:
> Should it be added to https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions?

Maybe it's just a question of where is the right place to add the conventions, but this particular location seems to imply that we are defining a file as a resource. I'm not opposed to this, but my understanding is that, in the context of the `file_input` operator / `filelog` receiver, we should not consider files to be resources. They are not the things that emitted the logs. They are essentially just a mechanism by which logs are transmitted.

So just to be clear, are we talking about defining file-related fields as a resource, but then just using the same convention to structure our attributes here? Or do we need to add a parallel section to the spec, which specifically defines attribute conventions for files?

Maybe this isn't an important nuance, but I want to make sure we're not missing it.

Reply:
First pass proposal for a new `file.*` namespace:
```
file.name
file.path
file.name.resolved
file.path.resolved
file.stream
```

Reply:
I think you are right, these are not resource attributes, they are log record attributes.